### PR TITLE
Priority: Added additional information/columns to CourseDetailsTable

### LINKED ISSUE #23: Priority: Add additional information to CourseDetailsTable
DEPENDENCY: #14 

Add all the additional fields (Course ID, Title, Enrolled, Location, Days, Time, Instructor) to the table for CourseDetails.
-Adding columns for the rest of the fields ensures that when the user accesses the Course Details page, the user can see all the information for that specific course. They will have access to the meeting days/times/location, the course ID an title, how many people are enrolled, and who will be teaching the course.
-Modify the tests for the CourseDetailsTable file as well and make sure it has full coverage.

### PR DESCRIPTION
**Pull request by rosillo-m**
In this PR, additional information was added to the table for course details. The purpose of adding this info to the table is to ensure the user can see all the details such as the title and course id, meeting information, and instructor of the course they have selected.

Here is a picture of how it looks in the storybook before changes in enrollmentCode:
Screenshot 2023-11-27 at 11 30 48 PM

Screenshot 2023-11-27 at 11 31 09 PM

After changes in enrollmentCode:
Screenshot 2023-11-28 at 2 35 21 PM

Screenshot 2023-11-28 at 2 35 43 PM

Here is the link to the storybook as well: https://ucsb-cs156-f23.github.io/proj-courses-f23-7pm-3/prs/34/storybook/?path=/docs/components-coursedetails-coursedetailstable--empty

Closes #23

### CODE REVIEW

**Comment by rosillo-m** [comment]
Professor @pconrad, I have updated the information in both the PR and the issue itself.

File: frontend/src/fixtures/courseDetailsFixtures.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const courseDetailsFixtures = {
+  twoCourses: [
+    {
+      quarter: "20211",
```
Comment (adyahrastogi) [inline frontend/src/fixtures/courseDetailsFixtures.js:4]: I think we are only looking at data from 2022 onwards (start quarter 20221), so maybe we can look at classes that are from 2022 onwards.

**Review by adyahrastogi — DISMISSED** [review]
Looks good to me, the format of the courses is fixed now!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**